### PR TITLE
Develop/fixes/TD-752 mobil view issues

### PR DESCRIPTION
**Pull request by OluwatobiAwe**
### JIRA link
[TD-752](https://hee-tis.atlassian.net/browse/TD-752)

### Description
Description which contained bulleted point data  provided below the ‘Competency’ heading for the self assessments  not aligned properly.

### Screenshots
_Attach screenshots on mobile, tablet and desktop._

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors (see [info on Text Editor settings](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546185813/DLS+Dev+Process) to avoid whitespace changes)
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [ ] Manually tested my work with and without JavaScript
- [ ] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546939432/DLS+Code) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

[TD-752]: https://hee-tis.atlassian.net/browse/TD-752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by martinfoster-hee — COMMENTED** [review]
That max-width: 48.0525em seems oddly specific - can that be pulled out to a scss constant to give it some context maybe?

File: DigitalLearningSolutions.Web/Styles/layout.scss [hunk 1]
```diff
@@ -343,3 +343,9 @@ nav, .nhsuk-header__navigation, #header-navigation {
     margin-top: 10px;
   }
 }
+
+@media (max-width: 48.0525em) {
```
Comment (martinfoster-hee) [inline DigitalLearningSolutions.Web/Styles/layout.scss:347]: That max-width: 48.0525em seems oddly specific - can that be pulled out to a scss constant to give it some context maybe?

Comment (OluwatobiAwe) [reply DigitalLearningSolutions.Web/Styles/layout.scss]: This has been updated.

Comment (martinfoster-hee) [reply DigitalLearningSolutions.Web/Styles/layout.scss]: Thanks @OluwatobiAwe Your 768px constant is a much more familar (non-magic) number thanks 👍

**Review by kevwhitt-hee — APPROVED** [review]

**Pull request merged**